src/ape/exceptions.py

```diff
@@ -47,8 +47,13 @@ class ArgumentsLengthError(ContractError):
     Raised when calling a contract method with the wrong number of arguments.
     """
 
-    def __init__(self):
-        message = "The number of the given arguments do not match what is defined in the ABI."
+    def __init__(self, arguments_length: Optional[int] = None, inputs_length: Optional[int] = None):
+        message = (
+            f"The number of the given arguments "
+            f"{f'({arguments_length}) ' if arguments_length else ''}"
+            f"do not match what is defined in the "
+            f"ABI{f' ({inputs_length})' if inputs_length else ''}."
```

Comment:
```suggestion
            "The number of the given arguments"
            f"{f' ({arguments_length})' if arguments_length else ' '}"
            " do not match what is defined in the "
            f" ABI{f' ({inputs_length})' if inputs_length else ''}."
```

Reply:
I will trust that you tested this and it works.